README.md

```diff
@@ -28,32 +28,24 @@ yarn prepare
 ```
 
 ### Database Setup 🐘
-0. Install Postgres (v14.5)
-- on OS X
-```sh
-brew install postgresql
-createuser -s postgres
-brew services restart postgresql
-```
 
-1. Init the database
-```sh
-./init-db.sh
-```
+For simplicity we provide a dockerized postgres. As a prerequisite, you will need to install 
+[Docker Desktop](https://www.docker.com/products/docker-desktop/).
```

Comment:
future ticket: thinking about a new contributor, perhaps we should make a prerequisites section that highlights different tools the  dev needs to install to contribute. (I think it's yarn 2, node 18, and docker desktop at this point 🤔 )